aten/src/ATen/native/native_functions.yaml

```diff
@@ -6185,12 +6185,12 @@
     CompositeExplicitAutogradNonFunctional: _nested_view_from_buffer_copy
   autogen: _nested_view_from_buffer_copy.out
 
-- func: _nested_view_from_jagged(Tensor(a) self, Tensor offsets, Tensor dummy, Tensor? lengths=None, int ragged_idx=1) -> Tensor(a)
+- func: _nested_view_from_jagged(Tensor(a) self, Tensor offsets, Tensor dummy, Tensor? lengths=None, int ragged_idx=1, Tensor? min_seqlen=None, Tensor? max_seqlen=None) -> Tensor(a)
```

Comment:
quick question, do we want to eventually make max/min_seqlen available from the torch.nested.nested_tensor_from_jagged API?

Reply:
We definitely need a way for the user to provide these manually. Adding them to the `nested_tensor_from_jagged()` API is one way to do this. One additional way / alternative is to provide public setter `@properties` for min / max sequence length. Some better Dynamo tracing support for `@properties` landed recently so this may be feasible now.

I was originally thinking we'd only allow manual setting through setter properties, but I could probably be convinced that the `nested_tensor_from_jagged()` API should accept these too. Opinions on this?